cfme/tests/test_utilization_metrics.py

```diff
@@ -92,7 +92,7 @@ def metrics_collection(appliance, clean_setup_provider, provider, enable_candu):
         elif provider.category == "infra" and vm_count and host_count:
             return
         else:
-            time.sleep(15)
+            time.sleep(20)
 
     if time.time() > start_time + timeout:
         raise Exception("Timed out waiting for metrics to be collected")
```

Comment:
I know we've had this code in metrics_collection() for sometime now and I think it's time to switch to wait_for :).

Reply:
As per @nachandr and my discussion; we going to refactor C&U codes; At that time we will add those changes.

Reply:
**REQUIRED:** Please add a TODO in there so we don't forget

Reply:
@psav added `ToDo: refactor code with wait_for`